tests/test_client.py

```diff
@@ -89,7 +89,7 @@ class TestTimeouts(unittest.TestCase):
     def test_request_timeout(self):
         with self.assertRaises(dwave.cloud.exceptions.RequestTimeout):
             with Client(request_timeout=0.00001, **config) as client:
-                client.get_solvers()
+                client.get_solvers(refresh=True)
```

Comment:
Why is a refresh required here but not elsewhere?

Reply:
Elsewhere we want to use the cache (to reduce runtime), and here we want to force an actual network request in order to test `request_timeout` is honored, and a `RequestTimeout` is raised if server doesn't respond in a timely manner (we use the unrealistic 10us to make sure request always fails).